docs/storage.md

```diff
@@ -112,4 +112,4 @@ You have to enable the Milvus backend by running, here is the [quickstart](https
 You can configure Milvus connection via command `memgpt configure`.
 ```sh
 ? Select storage backend for archival data: milvus
-? Enter the Milvus connection URI (Default: localhost:19530): localhost:19530
+? Enter the Milvus connection URI (Default: /Users/sarahwooders/.memgpt/milvus.db): /Users/sarahwooders/.memgpt/milvus.db
```

Comment:
can we remove this diff?

Reply:
What this diff do is to change the default URI from `IP:port` to a `local_file_path`. In the case of `IP:port`, user have to start a milvus serive from docker, while in case `local_file_path`, milvus lite service will start under the hood without any other manual operation. So that is convenient for users.

If the description of this path is inappropriate, can it be changed to other descriptions like:
- Enter the Milvus connection URI (Default: ~/.memgpt/milvus.db): ~/.memgpt/milvus.db
- Enter the Milvus connection URI (Default: [MEMGPT_DIR]/milvus.db): [MEMGPT_DIR]/milvus.db

Reply:
Let me clarify it in another  PR: https://github.com/cpacker/MemGPT/pull/1407